app/grandchallenge/evaluation/forms.py

```diff
@@ -124,8 +126,28 @@ class Meta:
             "extra_results_columns": JSONEditorWidget(
                 schema=EXTRA_RESULT_COLUMNS_SCHEMA
             ),
+            "submissions_open": forms.DateInput(
+                format=("%Y-%m-%d"), attrs={"type": "date"}
+            ),
+            "submissions_close": forms.DateInput(
+                format=("%Y-%m-%d"), attrs={"type": "date"}
+            ),
         }
 
+    def clean(self):
+        submissions_open = self.cleaned_data["submissions_open"]
+        submissions_close = self.cleaned_data["submissions_close"]
+
+        if (
+            submissions_open
+            and submissions_close
+            and not submissions_close > submissions_open
```

Comment:
Nitpick: easier to read if the bool is not negated: `and submissions_close < submissions_open`